src/pow.cpp

```diff
@@ -147,6 +219,41 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
     return bnNew.GetCompact();
 }
 
+/** Perform the fork difficulty reset */
+unsigned int CalculateMVFResetWorkRequired(const CBlockIndex* pindexLast, int64_t nFirstBlockTime, const Consensus::Params& params)
+{
+    const arith_uint256 bnPowLimit = UintToArith256(params.powLimit); // MVF-BU moved here
+
+    arith_uint256 bnNew, bnNew1, bnNew2, bnOld;
+
+    // drop difficulty via factor
+    int nDropFactor = 4;
+    // use same formula as standard
+    int64_t nActualTimespan = pindexLast->GetBlockTime() - nFirstBlockTime;
+    // used reduced target time span
+    int64_t nTargetTimespan = nActualTimespan / nDropFactor;
+
+    bnOld.SetCompact(pindexLast->nBits);
+    bnNew1 = bnOld / nTargetTimespan;
+    bnNew2 = bnNew1 * nActualTimespan;
+
+    // check for overflow or overlimit
+    if (bnNew2 / nActualTimespan != bnNew1 || bnNew2 > bnPowLimit)
+        bnNew = bnPowLimit;
+    else bnNew = bnNew2;
+
+    // ignore formula above and override with fixed difficulty
+    //bnNew.SetCompact(0x207eeeee);
```

Comment:
We can emulate any fixed difficulty using the factor, and I think we basically agree that the factor model is a good way forward.
So unless you think there are some technical obstacles, I think we can discard the fixed difficulty reset approach... keep the code cleaner makes it easier for us to work on :-)